samples/sensor/mcux_acmp/boards/mimxrt1170_evk_cm4.overlay

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
can these configurations move to the boards folder?

Reply:
It's kept aligned with twr_ke18f.overlay. Per my understanding, the acmp1 in2 is only used for the sample test, there may be pin conflicts with other modules and there's no obviously fixed map of comparator on the board.